gocourse05/server/log_sevice.go

```diff
@@ -0,0 +1,31 @@
+package server
+
+import (
+	"os"
+	"time"
+)
+
+type LogService struct {
+	strangeApi string // idk just to be...
+}
+
+func NewLogService(strangeApi string) *LogService {
+	return &LogService{
+		strangeApi: strangeApi,
+	}
+}
+
+func (a *LogService) SendData(s string) {
```

Comment:
Замість паніки краще повертати `error`:
```suggestion
func (a *LogService) SendData(s string) error {
```